docs/animations/Saturation.md

```diff
@@ -1,14 +1,13 @@
 ---
 title: Saturation animation behavior
 author: nmetulev
-ms.date: 08/20/2017
 description: The Saturation animation behavior selectively saturates a XAML element.
 keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, saturation animation, saturation
 ---
 
 # Saturation
 
-The Saturation animation selectively saturates a XAML element. Saturation animation is applied to all the XAML elements in its parent control/panel. Saturation animation doesn't affect the functionality of the control.
+The [Saturation animation](https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.uwp.ui.animations.animationextensions) selectively saturates a XAML element. Saturation animation is applied to all the XAML elements in its parent control/panel. Saturation animation doesn't affect the functionality of the control.
```

Comment:
Add id to link #Microsoft_Toolkit_Uwp_UI_Animations_AnimationExtensions_Saturation_